Automattic/jetpack
# Forms: Forms editor style issue

## Proposed changes:

* Fix Forms editor style issue introduced by #28973

Before
![image](https://user-images.githubusercontent.com/7811225/219465613-23339082-5124-4a20-8ceb-5ce0e62970e3.png)

After
![image](https://user-images.githubusercontent.com/7811225/219465697-7a3e3e1c-b71a-46dc-8c7d-485fe5a1ad1d.png)


### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Create a post/page and add a Form block
* Select a variation
* Check if the fields style, especially the Multiline Text Field and the Dropdown field, look good
